docs/How-To-Use-RxJava.md

```diff
@@ -11,22 +11,21 @@ You can find additional code examples in the `/src/examples` folders of each [la
 ### Java
 
 ```java
-public static void hello(String... names) {
-    Observable.from(names).subscribe(new Action1<String>() {
-
-        @Override
-        public void call(String s) {
-            System.out.println("Hello " + s + "!");
-        }
-
-    });
+public static void main(String[] args) {
```

Comment:
No, I meant show codes as anonym inner class and lambda, but keep the essence of the original example, i.e. fromArray

Reply:
Oh ok, i'll fix.